ros_ign_gazebo_demos/package.xml

```diff
@@ -7,7 +7,11 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <depend>ignition-gazebo3</depend>
+  <!-- Blueprint -->
+  <depend condition="$IGNITION_VERSION == blueprint">ignition-gazebo2</depend>
```

Comment:
I think this could be a `<exec_depend>`.

Reply:
Good point 6a24b28